src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -424,8 +430,14 @@ public class InstancesController : ControllerBase
             {
                 // start process and goto next task
                 instanceTemplate.Process = null;
-                string startEvent = await _altinnApp.OnInstantiateGetStartEvent();
-                processResult = _processService.ProcessStartAndGotoNextTask(instanceTemplate, startEvent, User);
+
+                ProcessChangeContext processChangeContext = new ProcessChangeContext();
+                processChangeContext.Instance = instanceTemplate;
+                processChangeContext.User = User;
+                processChangeContext.Prefill = instansiationInstance.Prefill;
+                processChangeContext.DontUpdateProcessAndDispatchEvents = true;
```

Comment:
What is the scenario for when we don't won't to update the process and dispatch events?

Reply:
@RonnyB71 it is instansiation. Then we create instance, but possible have form from input or form from copy. In that case we don't that startup to create missing dataelements.